doajtest/unit/test_upgrade.py

```diff
@@ -0,0 +1,78 @@
+import json
+import time
+import re
+from collections import OrderedDict
+
+from doajtest.helpers import DoajTestCase
+from portality import models
+from portality.upgrade import do_upgrade
+
+def operation(journal):
+    j = models.Journal.pull(journal.id)
+    bj = j.bibjson()
+    bj.title = "Updated Title"
+    j.save()
+    return j
+
+
+class TestUpgrade(DoajTestCase):
+
+    def test_upgrade(self):
+        # populate the index with some journals with title
+        saved_journals = []
+        for i in range(5):
+            j = models.Journal()
+            j.set_in_doaj(True)
+            bj = j.bibjson()
+            bj.title = "Test Journal"
+            bj.add_identifier(bj.P_ISSN, "{x}000-0000".format(x=i))
+            bj.publisher = "Test Publisher {x}".format(x=i)
+            bj.add_url("http://homepage.com/{x}".format(x=i), "homepage")
+            j.save()
+            saved_journals.append(j.id)
+
+        # and with some journals without title
+        for i in range(5):
+            j = models.Journal()
+            j.set_in_doaj(True)
+            bj = j.bibjson()
+            bj.add_identifier(bj.P_ISSN, "{x}000-0001".format(x=i))
+            bj.title = "Journal to Change"
+            bj.publisher = "Test Publisher {x}".format(x=i)
+            bj.add_url("http://homepage.com/{x}".format(x=i), "homepage")
+            j.save()
+            saved_journals.append(j.id)
+
+        created = j.last_updated
```

Comment:
`created` will only have the last_updated value of the very last of the 10 journals. Is that correct for the purposes of this test?